packages/devtools_app/lib/src/memory/memory_heap_tree_view.dart

```diff
@@ -520,33 +577,63 @@ class HeapTreeViewState extends State<HeapTree>
             size: defaultIconSize,
           ),
         ),
-        const SizedBox(width: defaultSpacing),
+      ],
+    );
+  }
+
+  Widget _buildAllocationsControls(ThemeData themeData) {
+    return Row(
+      children: [
         FixedHeightOutlinedButton(
           buttonKey: allocationMonitorKey,
-          tooltip: 'Monitor Allocations',
+          tooltip: 'Collect Allocation Statistics',
           onPressed: () async {
             MemoryScreen.gaAction(key: allocationMonitorKey);
             await _allocationStart();
           },
-          child: createImageIcon(
-            // TODO(terry): Could we use a canned material icon check w/ Youyang?
-            'icons/memory/communities_white@2x.png',
-            size: defaultIconThemeSize,
+          child: Row(
+            children: [
+              createImageIcon(
+                // TODO(terry): Match shape in event pane.
+                themeData.isDarkTheme
+                    ? 'icons/memory/communities_white@2x.png'
+                    : 'icons/memory/communities_black@2x.png',
+                size: defaultIconThemeSize,
+              ),
+              const SizedBox(width: denseSpacing),
+              const Text('Track')
+            ],
           ),
         ),
+        const SizedBox(width: denseSpacing),
         FixedHeightOutlinedButton(
```

Comment:
change material icon label to accept a Widget for icon instead of IconData. Then you can use that as the child of FixedHeightOutlinedButton instead of creating your own Row. This will ensure consistency with other buttons in DevTools in terms of icon size and label padding

Reply:
I'll do in another PR added TODO.